scripts/cibuild.sh

```diff
@@ -0,0 +1,134 @@
+        set +e
+        docker-compose -f "${DIR}/../docker-compose.yml" exec -T postgres gosu postgres psql -d rasterfoundry -c 'select 1' >/dev/null 2>/dev/null
+        status_check=$?
+        if [ $status_check == 0 ]
+        then
+            echo "Connected to database successfully"
+            break
+        fi
+        set -e
+        if [[ ${counter} == "${max}" ]]
+        then
+            echo "Could not connect to database after some time"
+            exit 1
+        fi
+        sleep 3
+        (( counter++ ))
+    done
+}
+
+
+function create_airflow_database() {
+    docker-compose -f "${DIR}/../docker-compose.yml" up -d postgres
+    check_database
+    set +e
+    # Create database (have to ignore errors if database already exists)
+    docker-compose -f "${DIR}/../docker-compose.yml" \
+        exec -T postgres gosu postgres createdb airflow >/dev/null 2>/dev/null
+    set -e
+    GIT_COMMIT=${GIT_COMMIT} docker-compose \
+        -f "${DIR}/../docker-compose.yml" \
+        run --rm airflow-webserver airflow initdb
+
+    docker-compose -f "${DIR}/../docker-compose.yml" stop postgres
+}
+
+function setup() {
+
+    # Pull additional services
+    docker-compose \
+        -f "${DIR}/../docker-compose.yml" \
+        pull nginx postgres swagger-editor redis
+
+    create_airflow_database
+
+    # app-server updates
+    docker-compose \
+        -f "${DIR}/../docker-compose.yml" \
+        run --rm app-server sbt update
+
+    # app-frontend updates
+    docker-compose \
+        -f "${DIR}/../docker-compose.yml" \
+        run --rm app-frontend npm install --quiet
+
+    docker-compose \
+        -f "${DIR}/../docker-compose.yml" \
+        run --rm app-frontend npm run build
+}
+
+
+if [[ -n "${GIT_COMMIT}" ]]; then
+    GIT_COMMIT="${GIT_COMMIT:0:7}"
+else
+    GIT_COMMIT="$(git rev-parse --short HEAD)"
+fi
+
+if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
+    if [ "${1:-}" = "--help" ]; then
+        usage
+    else
+        echo "Pulling down testing environment..."
+        pushd "${DIR}/.."
+        # Download environment configuration from S3
+        aws s3 cp "s3://rasterfoundry-testing-config-us-east-1/.env" ".env"
+        popd    
+
+        # Build/pull containers
+        echo "Building/Pulling containers..."
+        setup
+
+        ./scripts/test
+
+        # Clean up containers after testing
+        echo "Cleaning up containers..."
+        docker-compose stop
+        docker-compose rm -f --all
+
+        # Build JAR file
+        echo "Building JAR file..."
+        GIT_COMMIT=${GIT_COMMIT} docker-compose \
+            run --no-deps --rm --entrypoint ./sbt app-server app/assembly
+        
+        # Build app, airflow
+        echo "Building app, airflow..."
+        GIT_COMMIT=${GIT_COMMIT} docker-compose \
+            -f "${DIR}/../docker-compose.yml" \
+            -f "${DIR}/../docker-compose.test.yml"\
+            build
+
+
+        # If the cibuild.d directory exists, source each script it
```

Comment:
I don't think this functionality needs to be brought in here until we have a reason to hook into it.